pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -138,13 +124,15 @@ buildPythonPackage rec {
     find "$IBIS_TEST_DATA_DIRECTORY" -type f -exec chmod u+rw {} +
 
     # load data
-    for backend in ${backendsString}; do
-      python ci/datamgr.py "$backend" &
+    for backend in ${lib.concatStringsSep " " backends}; do
+      python ci/datamgr.py load "$backend"
     done
 
-    wait
-  '' + lib.optionalString (lib.versionOlder version "3.0.0") ''
-    export PYTEST_BACKENDS="${backendsString}"
+    python -c 'import ibis; assert ibis.__version__ == "${version}"'
```

Comment:
Why do we assert the version here? We are downloading from a tag above so I don't think we need this.

Reply:
Removed.